### PR TITLE
proxy-status: ref RFC 8941 and add missing parentheses

### PR DESCRIPTION
**Pull request by reschke**

### CODE REVIEW

File: draft-ietf-httpbis-proxy-status.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ Note that in this specification, "proxy" is used to indicate both forward and re
 
 The Proxy-Status HTTP response field allows an intermediary to convey additional information about its handling of a response and its associated request.
 
-It is a List {{I-D.ietf-httpbis-header-structure, Section 3.1}}:
+It is a List ({{I-D.ietf-httpbis-header-structure, Section 3.1}}):
```
Comment (reschke) [inline draft-ietf-httpbis-proxy-status.md:78]:
```suggestion
It is a List ({{RFC8941, Section 3.1}}):
```

**Comment by mnot** [comment]
You need to put the RFC reference in the normative section...

**Comment by reschke** [comment]
Feel free to discard the PR and just do the right thing :-).

**Comment by mnot** [comment]
OBE

**Pull request closed without merging**